lib/kb_cobrapy/Utils.py

```diff
@@ -36,13 +39,29 @@ def validate_params(params, expected, opt_param=set()):
     def _ws_obj_to_cobra(self, ref):
         ret = self.dfu.get_objects({'object_refs': [ref]})['data'][0]
         name = ret['info'][1]
-        model = cobrakbase.convert_kmodel(ret['data'])
-
+        
+        #old nasty method
+        #model = cobrakbase.convert_kmodel(ret['data'])
```

Comment:
unused code can be removed.

Reply:
I'll leave it for this version, just for easy revert